README.md

```diff
@@ -1,5 +1,28 @@
-# Welcome To Kurtosis
+# Kurtosis Quickstart - TL;DR Version
 
-1. `main.star` contains a boiler plate Kurtosis Script
-2. Run it using `kurtosis run main.star` in the terminal
-3. Head to our [quickstart](https://docs.kurtosis.com/quickstart) for more
\ No newline at end of file
+The [original quickstart][quickstart] in our docs is in a "code-along" format where folks are expected to follow and run the commands and actions on their local machine. If you'd prefer to dive straight into running the end results and exploring the code to learn about Kurtosis, then this TL;DR version of the quickstart is for you.
+
+### To get straight to the end result, simply run:
+```bash
+kurtosis run --enclave quickstart main.star
+```
+
+The `main.star` file in this directory contains the build steps for a composable and portable multi-container test environment. Specifically, Kurtosis will use those instructions to:
+- [Start a containerized Postgres database in Kurtosis][run-postgres],
+- [Seed your database with test data using task sequencing][add-data],
+- [Connect an API server to your database using dynamic service dependencies][add-an-api], and
+- [Parameterize your application setup in order to automate loading data into your API][insert-data]
+
+Feel free, at any time, to refer to the [conclusion][conclusion] and the other review sections to get an explanation of the concepts and capabilities of Kurtosis.
+
+### To run tests against your test environment:
```

Comment:
might have to add language support for golang in the Gitpod but we can do that after we merge this too


broadly speaking - Do we want all of this here? Can it just be Starlark?

Reply:
We've heard from users that _if_ we are claiming to be a tool for "test environments", then we should very explicitly show them how to run a test. I think that makes a lot of sense.

Reply:
Merging this and fixing this without any guardrails on.